examples/bankofanthos/transactionhistory/transactionhistory.go

```diff
@@ -44,7 +44,6 @@ type impl struct {
 	ledgerReader *common.LedgerReader
 }
 
-var _ T = (*impl)(nil)
```

Comment:
I'm a bit confused about this piece of code, I thought the validation code here would be auto-generated in `weaver_gen.go`.

Reply:
Yup, you're right!